tests/packagedcode/data/about/apipkg.ABOUT-expected

```diff
@@ -39,17 +39,17 @@
         "license_expression_spdx": "MIT",
         "matches": [
           {
-            "score": 100.0,
+            "license_expression": "mit",
+            "spdx_license_expression": "MIT",
+            "from_file": null,
```

Comment:
@swastkk there are still loads of files with updates like these (just changing the order of attributes, nothing related to this PR). Please delete the commit updating the test expectations, run tests and only regenerate the tests which have failed. See comment above on this.

Reply:
Resolved